text/0073-referedum-deposit-track.md

```diff
@@ -0,0 +1,70 @@
+# RFC-0073: Decision Deposit Referendum Track
+
+|                 |                                                                                |
+| --------------- |--------------------------------------------------------------------------------|
+| **Start Date**  | 12 February 2024                                                               |
+| **Description** | Add a referendum track which can place the decision deposit on any other track |
+| **Authors**     | JelliedOwl                                                                     |
+
+## Summary
+
+The current size of the decision deposit on some tracks is too high for many proposers. As a result, those needing to use it have to find someone else willing to put up the deposit for them - and a number of legitimate attempts to use the root track have timed out. This track would provide a more affordable (though slower) route for these holders to use the root track.
+
+## Motivation
+
+There have been recent attempts to use the Kusama root track which have timed out with no decision deposit placed. Usually, these referenda have been related to parachain registration related issues. 
+
+## Explanation
+
+Propose to address this by adding a new referendum track ***[22] Referendum Deposit*** which can place the decision deposit on another referendum. This would require the following changes:
```

Comment:
What you are asking in fact is to just reduce the deposit to 1000 DOT.  
The fact that this is done through a new track does not matter i think since the Treasury does not have a counterparty risk here from being slashed.  
What i mean: if 100,000 DOT from the treasury is slashed, then nobody cares.

Reply:
It's not quite the same since the number of deciding referenda on the root track is small (I think), so I definitely wouldn't be advocating for reducing the DD there without also increasing the number which could in deciding at any one time.

Reply:
> The fact that this is done through a new track does not matter

I must respectfully disagree. Placing a decision deposit directly on a referendum is *permissionless* whereas this RFC proposes a *permissioned* way to place a decision deposit via OpenGov.

A referendum that seeks to place a decision deposit is fundamentally different from a referendum that seeks to execute an action. Voting in favor of the former signals, "I want this referendum to be decided on by token holders without the disincentive of slashing" whereas voting in favor of the latter signals, "I want this action to be executed on chain." These are two very different statements. If a majority of token holders agree on the former, why shouldn't that be allowed? Isn't this supposed to be *Open*Gov?

The current status quo implies that no referendum should be decided on unless the decision deposit comes from some account permissionlessly. I think that is quite inflexible, especially since there are well-intentioned individuals that should be given recourse, other than the Fellowship, for placing large decision deposits.

Reply:
Maybe this is the wrong place to decide it - the Fellowship should be consulted for technical questions - not political ones. I suggest bringing it up here ASAP: https://github.com/polkadot-fellows/runtimes/pull/184